test/Dockerfile

```diff
@@ -12,21 +14,31 @@ RUN apt-get update && apt-get install -y \
   libfdisk-dev \
   libnl-genl-3-dev
 
+# Required for building the cgi example
```

Comment:
Shouldn't this be a separate change?

Reply:
No, as this is only necessary due to moving to explicitly installing dependencies (`--no-install-recommends`).